dev_guide/copy_files_to_container.adoc

```diff
@@ -12,7 +12,7 @@ toc::[]
 
 == Overview
 
-You can use the CLI to copy local files to or from a remote directory in a container.  This is a useful tool for copying database archives to and from your pods for backup and restore purposes.  It can also be used to copy source code changes into a running pod for development debugging, when the running pod supports hot reload of source files.
+You can use the CLI to copy local files to or from a remote directory in a container.  This is a useful tool for copying database archives to and from your pods for backup and restore purposes.  Note that a database pod's persistent volume directory is 'var/lib/[mysql|mongodb|pgsql]/data'.  This tool can also be used to copy source code changes into a running pod for development debugging, when the running pod supports hot reload of source files.
```

Comment:
path should be absolute (/var/lib/......) but i'm not sure we want to specify this here anyway.  There could be other database images that use other paths, this only covers the images we provide.  Doc about what path specific images use for volumes belongs w/ the image specific documentation.